app/presenters/chat_channel_detail_presenter.rb

```diff
@@ -1,10 +1,11 @@
 class ChatChannelDetailPresenter
```

Comment:
I didn't notice this pattern in the previous review but I wonder if we shouldn't remove presenters and just use jbuilder for rendering. They seem to be just classes used to serialize to JSON.

See https://github.com/thepracticaldev/dev.to/blob/master/app/views/articles/index.json.jbuilder for example

Reply:
To build the response for the API this will help to build response according to our use case.

Reply:
Sure, but I think it can be done directly in jbuilder as it's essentially a class that serializes to JSON. I'm just trying not to introduce too many different patterns in a code base that already has too many :)

Reply:
done